includes/definitions/konica.yaml

```diff
@@ -6,9 +6,9 @@ over:
     - { graph: device_toner, text: Toner }
 discovery:
     - sysDescr:
-        - 'KONICA MINOLTA'
+        - 'konica minolta'
```

Comment:
Hi @andreasmalta 
Could you add instead of replace the current value ? In case some device is using the capital version, to ensure it is not dropped after this PR is merged.
Thanx